cluster/network-plugins/cni/to_docker/bin/c2d-inner

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+if [ $# -ne 2 ]; then
+   echo Usage: $0 INPUT_FILE_NAME OUTPUT_FILE_NAME >& 2
+   exit 1
+fi
+INPFN="$1"
+OUTFN="$2"
+
+if [ "${DEBUG:-}" == "true" ]; then
+    echo
+    printenv | grep CNI
+    set -x
+fi
+
+set -e
+
+case "${CNI_COMMAND}" in
+    (ADD)
+        thenet="$(jq -r .name < "${INPFN}")"
```

Comment:
You mean you doubt it is correct for the inner dquotes to not be escaped?  Strange but true!  For example:
```
mspreitz@mvespa1-2:~$ y="a    b"
mspreitz@mvespa1-2:~$ x="$(echo "${y}")"
mspreitz@mvespa1-2:~$ echo "$x"
a    b
```